Frontend/src/pages/registration_administration/components/RegistrationAdministrationList.jsx

```diff
@@ -1,12 +1,12 @@
+import { useQuery } from '@tanstack/react-query'
 import { FlagIcon } from '@thewca/wca-components'
-import React, { useEffect, useMemo, useReducer, useState } from 'react'
+import React, { useMemo, useReducer } from 'react'
 import { Link, useParams } from 'react-router-dom'
 import { Checkbox, Popup, Table } from 'semantic-ui-react'
-import { getAllRegistrations } from '../../../api/registration/get/get_registrations'
-import getCompetitorInfo from '../../../api/user/get/get_user_info'
 import LoadingMessage from '../../../ui/messages/loadingMessage'
 import styles from './list.module.scss'
 import RegistrationActions from './RegistrationActions'
+import { getAllRegistrations } from '../../../api/registration/get/get_registrations'
 
 // Currently it is at the developer's discretion to make sure
 // an attendee is added to the right list.
```

Comment:
In theory this is slightly more problematic now. `useQuery` will automatically cache query results, and refetch them periodically (and when the page is refocused, etc). (These things can be configured on a query-by-query basis, and possibly globally.) Anyway, if a waiting list person is selected, then the query refetches and that person has since been moved to the accepted list by another organizer on a different computer, then the `selected` state here will still (incorrectly) say that the person is checked on the waiting list, but the UI will (correctly) have updated to show them in the accepted list.

In practice, not sure how much of an issue this will be, and the reducer was going to be revisited later anyway. So probably no need to do anything about it in this PR.

But the general fact that `useQuery` automatically refetches data at periodic/specific times is something to keep in mind, especially in forms.

Reply:
Should be fixed by useMutation in the late PR